src/vos/tests/vts_mvcc.c

```diff
@@ -1424,9 +1424,10 @@ uncertainty_check_exec_one(struct io_test_args *arg, int i, int j, bool empty,
 		daos_epoch_t	pe = ae - 1;
 
 		memcpy(pp, wp, strlen(wp));
-		print_message("  update(%s, "DF_U64") (expect 0): ", pp, pe);
+		print_message("  update(%s, "DF_U64") (expect DER_SUCCESS): ",
```

Comment:
I wish we could remove DER_SUCCESS and just use 0 consistently everywhere. But never mind.